stdlib/src/collections/dict.mojo

```diff
@@ -45,6 +45,14 @@ trait KeyElement(CollectionElement, Hashable, EqualityComparable):
     pass
 
 
+trait StringableKeyElement(KeyElement, Stringable):
+    pass
+
+
+trait StringableCollectionElement(CollectionElement, Stringable):
```

Comment:
**Suggestion** Let's move this right by `CollectionElement` and remove it from here and `list.mojo` - WDYT?  I'd prefer having a common spot for it rather than the trait defined twice (once in `list.mojo` and once here).

Reply:
Sure thing

Reply:
Done